src/mock-doc/element.ts

```diff
@@ -372,6 +372,7 @@ export class MockCanvasElement extends MockHTMLElement {
           data: new Array(w * h * 4),
         };
       },
+      toDataURL() {},
```

Comment:
I think this should be hoisted to be a method on `MockCanvasMethod`, rather than the object returned by `getContext` - it appears that `toDataURL` is an instance method on a Canvas element according to https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL (please let me know if my understanding is incorrect though!).